src/linear_algebra/char_poly/coeff.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2020 Aaron Anderson, Jalex Stark. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson, Jalex Stark.
+-/
+
+import linear_algebra.char_poly
+import linear_algebra.matrix
+import ring_theory.polynomial.basic
+import algebra.polynomial.big_operators
+
+/-!
+# Characteristic polynomials
+
+We give methods for computing coefficients of the characteristic polynomial.
+
+## Main definitions
+
+- `degree_char_poly_eq_dim`
+- `det_from_char_poly`
+- `trace_from_char_poly`
```

Comment:
Might as well give the (informal) statements. The 2nd and 3rd currently require you to know the maths in order to guess their statements.